playbooks/upload_file_to_s3.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Upload a local file to S3
+  hosts: localhost
```

Comment:
This playbook needs to be run from a remote host, as the user isn't expected to have the files they want to upload present in the EE. The documentation should explain that an inventory must be configured to use this playbook and which host/group to use.

Reply:
@gravesm 
Updates: To avoid configuring all remote hosts with the AWS Python libraries, the controller performs the PUT operation on the S3 bucket, and the file to upload is expected to be on the remote host.
Tell me if this makes sense for you